src/cgi.h

```diff
@@ -49,8 +49,8 @@ extern int cgi_display_errors;
 extern void cgi_init_headers(void);
 extern void cgi_redirect(char *url);
 extern void cgi_fatal(const char *error);
```

Comment:
The following could be problematic, because it changes the API of the library. First ist change from char to const char, which should be fine. Second is a change from signed to unsigned. Both could at least produce warnings with existing code.